src/Application/Commands/HandleOutOfSyncFunds.php

```diff
@@ -74,8 +77,17 @@ protected function doExecute(InputInterface $input, OutputInterface $output): in
         $numFundingsCorrect = 0;
         $numFundingsOvermatched = 0;
         $numFundingsUndermatched = 0;
+
+        $problemIds = [];
         /** @var CampaignFunding[] $fundings */
-        $fundings = $this->campaignFundingRepository->findAll();
+        if ($mode === '2024-06-05-patch') {
+            $problemIds = [29822];
+            $output->writeln('Running in 2024-06-05-patch mode');
+            // https://stackoverflow.com/a/52427915/2803757
+            $fundings = $this->campaignFundingRepository->findBy(['id' => $problemIds]);
```

Comment:
@NoelLH are you sure findBy works with an array of ids rather than a single ID?

We could simplify this line to
`$this->campaignFundingRepository->find(29822);`

Reply:
Looks like this is good. Hard to find actual docs on what criteria can be for findBy, but the implementation isn't that hard to follow, it does test if the $problemIds is an array and generate an SQL `IN` query if so:

![image](https://github.com/thebiggive/matchbot/assets/159481/2e125409-46f0-4c51-8a0f-eb488160128c)

Reply:
Yep, I think I vaguely remember checking this last time – and moreover we saw it work live then. But definitely good to double check, thanks!